quakemigrate/lut/create_lut.py

```diff
@@ -230,7 +259,7 @@ def _compute_1d_fmm(lut, phase, vmodel):
 
     Parameters
     ----------
-    lut : :class:`~quakemigrate.lut.LUT` object
+    lut : :class:`~quakemigrate.lut.lut.LUT` object
```

Comment:
Yeah, this is something I was unsure of when initially adding many of these - I think either is fine, so long as we are consistent. I used the former representation as, for example, `numpy.ndarray` is used very commonly, despite the actual class definition being under `numpy.core`. On the other hand, much of the obspy documentation provides full paths.

Reply:
For local functions, readthedocs doesn't successfully make a link unless you use the full version (have experimented). (e.g. look at the Return type: LUT object [here](https://quakemigrate.readthedocs.io/en/latest/submodules/quakemigrate.lut.html#quakemigrate.lut.create_lut.compute_traveltimes))

Reply:
Great, then that settles it. Worth doing a package-wide sweep?

Reply:
Probably. I've been trying to do it in each file I've edited over the past couple of months, but there will defo be some still loitering about.

Reply:
(Done)